test/integration/scheduler/queue_test.go

```diff
@@ -542,6 +576,24 @@ func TestCoreResourceEnqueue(t *testing.T) {
 					}
 				}
 
+				for _, pvc := range tt.initialPvcs {
+					volType := v1.HostPathDirectoryOrCreate
+					_, err := testutils.CreatePV(cs, st.MakePersistentVolume().
+						Name(pvc.Spec.VolumeName).
+						AccessModes([]v1.PersistentVolumeAccessMode{v1.ReadOnlyMany}).
+						Capacity(pvc.Spec.Resources.Requests).
+						HostPathVolumeSource(&v1.HostPathVolumeSource{Path: "/tmp", Type: &volType}).
+						Obj())
+					if err != nil {
+						t.Fatalf("cannot create pv: %v", err)
```

Comment:
To make it consistent with the next log:
```suggestion
						t.Fatalf("Failed to create PV %q: %v", pvc.Spec.VolumeName, err)
```